### PR TITLE
added claude cost per token

### PR DESCRIPTION
**Pull request by WilliamEspegren**

### CODE REVIEW

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -112,5 +112,25 @@
     "together-ai-40.1b-70b": {
         "input_cost_per_token": 0.000003,
         "output_cost_per_token": 0.000003
-    }
+    },
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json:115]: we already have claude see line 42

Comment (ishaan-jaff) [reply model_prices_and_context_window.json:115]: we're missing some vertex llms tho 

I'd recommend seeing llms in __init__.py to see what's missing

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]
thanks for the PR ! we already have claude in the model_prices_context_window.json. but feel free to add other llms

**Comment by WilliamEspegren** [comment]
Woops, my bad 😅

**Pull request closed without merging**